example/AndroidManifest.xml

```diff
@@ -11,7 +11,6 @@
         android:label="@string/appName">
         <activity
             android:name=".activity.PaymentActivity"
-            android:screenOrientation="portrait"
```

Comment:
Is the user responsible for maintaining state and re-loading on re-create onLayoutChange?